app/controllers/report_controller/schedules.rb

```diff
@@ -241,7 +241,7 @@ def schedule_edit
       add_flash(_("All changes have been reset"), :warning) if params[:button] == "reset"
       if x_active_tree != :reports_tree
         # dont set these if new schedule is being added from a report show screen
-        obj = find_checked_ids_with_rbac(MiqSchedule)
+        obj = find_records_with_rbac(MiqSchedule, checked_or_params)
```

Comment:
There is a difference between the returned value of methods `find_checked_ids_with_rbac` and `find_records_with_rbac` (_id_ vs _instance_)